apps/frontend/src/components/SendForm.svelte

```diff
@@ -34,15 +43,19 @@
 </script>
 
 <form on:submit={handleSubmit}>
-  {#if chain === Chain.NEAR}
-    <SendNearFormPart bind:createTX />
-    <div class="flex flex-row justify-around pt-3">
-      {#if onCancel !== undefined}
-        <Button onClick={onCancel}>Cancel</Button>
-      {/if}
-      <Button type="submit">Submit</Button>
-    </div>
-  {:else}
-    Please enter a valid chain through which to send funds
-  {/if}
+  <!-- TODO: bind createTX somehow!! -->
```

Comment:
Add issue for this

Reply:
Woops, this is done, I need to remove comment